telemetry/definitions/commonDefinitions.json

```diff
@@ -2544,6 +2609,51 @@
             ],
             "passive": true
         },
+        {
+            "name": "awsq_serviceInvocation",
+            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
+            "metadata": [
+                {
+                    "type": "awsqAssignClickCount"
+                },
+                {
+                    "type": "awsqGenerateCodeIterationCount"
+                },
+                {
+                    "type": "awsqGenerateCodeClickCount"
+                },
+                {
+                    "type": "awsqApproachIterationCount"
+                },
+                {
+                    "type": "awsqIsRegeneratedCode"
+                },
+                {
+                    "type": "awsqFilesReviewedCount"
+                },
+                {
+                    "type": "awsqIsAcceptedCodeChanges"
```

Comment:
For this one and isApproachAccepted do we need the "Is" part?

Reply:
booleans in the file start with `is` or `has` don't think there is a rule for that.
@justinmk3 wdyt?